test/tla/cli-integration-tests.md

```diff
@@ -1050,21 +1050,30 @@ EXITCODE: OK
 ### check Bug914 succeeds
 
 Regression test for https://github.com/informalsystems/apalache/issues/914
-In the earlier version, we expected the model checker to complain about mismatching
-record types. In the latest version, this bug disappeared, due to the changes
-in the type checker.
+In the earlier version, we expected the model checker to complain about mismatching record types. In the latest version,
+this bug disappeared, due to the changes in the type checker.
 
 ```sh
 $ apalache-mc check Bug914.tla | sed 's/I@.*//'
 ...
 EXITCODE: OK
 ```
 
+### check RecordExcept987 succeeds
+
+Regression test for https://github.com/informalsystems/apalache/issues/987
+We should always use sorted keys on record types.
+
+```sh
+$ apalache-mc check Bug987.tla | sed 's/I@.*//'
+...
+EXITCODE: OK
+```
+
 ## configure the check command
 
-Testing various flags that are set via command-line options and the TLC
-configuration file. The CLI has priority over the TLC config. So we have to
-test that it all works together.
+Testing various flags that are set via command-line options and the TLC configuration file. The CLI has priority over
+the TLC config. So we have to test that it all works together.
```

Comment:
Any idea what's up with these formatting changes? The shorter wrapped lines are nicer imo,

Reply:
True. Don't know what happened